package.json

```diff
@@ -14,6 +14,7 @@
         "prettier": "^1.18.2",
         "prop-types": "^15.7.2",
         "query-string": "^6.5.0",
+        "react-redux": "^7.1.1",
```

Comment:
Is it ok to have this here and in peer dependencies? (I don't really know how they work...)

Reply:
yeah... annoyingly storybook was complaining about not having an instance of react-redux to play with. The only workaround I could find for now was installing it in here as well. 

Good spot though - this does cause problems for consuming projects which needs to be addressed in their webpack config aliases... technically making this a breaking change. I'll see if I can find a workaround before deploying.